apps/docs/src/app/core/component-docs/shellbar/examples/shellbar-collapsible-example.component.ts

```diff
@@ -4,7 +4,14 @@ import { ShellbarMenuItem, ShellbarUser, ShellbarUserMenu, ShellbarSizes } from
 
 @Component({
     selector: 'fd-shellbar-collapsible-example',
-    templateUrl: './shellbar-collapsible-example.component.html'
+    templateUrl: './shellbar-collapsible-example.component.html',
+    styles: [
+        `
+            fd-combobox {
+                min-width: 235px;
```

Comment:
Where is this value coming from? It should probably be in fund-styles.

Reply:
I'm not sure about this change, reverted it back for now.
If it will be considered needed let's handle it in separate issue